packages/cli/src/utils/generate.ts

```diff
@@ -141,13 +147,27 @@ function mergeCMSFiles() {
   mergeCMSFile('sections.json')
 }
 
+function createNodeModulesSymbolicLink() {
+  try {
+    fse.symlinkSync(repoNodeModulesDir, tmpDirNodeModulesDir)
+  } catch (err) {
+    console.error(err)
+  }
+
+  console.log('node_modules symbolic link created')
```

Comment:
How about adding a `from: ... to: ...` here? Would that be too verbose or actually useful?

Reply:
I plan to review all those logs once these PRs are merged. I don't think it makes sense to show them unless the `--verbose` option is enabled. I agree it would be useful to add `from: ... to: ...` here.